src/Illuminate/Support/Str.php

```diff
@@ -356,7 +356,12 @@ public static function singular($value)
      */
     public static function slug($title, $separator = '-')
     {
-        $title = static::ascii($title);
+        // Convert to UTF8
+        if (function_exists('mb_convert_encoding')) {
```

Comment:
You don't need this check. This function ALWAYS exists because we require the mb-string function.